backend/scheduler/tasks.py

```diff
@@ -57,13 +59,24 @@ def execute_pipeline_task(
     name: Any,
 ) -> None:
     logger.info(f"Executing pipeline name: {name}")
-
     try:
         logger.info(f"Executing workflow id: {workflow_id}")
         tenant: Organization = (
             get_tenant_model().objects.filter(schema_name=org_schema).first()
         )
         with tenant_context(tenant):
+            subscription_plugin_available = (
+                os.environ.get("SUBSCRIPTION_PLUGIN_AVAILABLE", "False") == "True"
+            )
+            logger.info(f"Is subscription available: {subscription_plugin_available}")
+
+            if subscription_plugin_available and not etl_prerun_check(org_schema):
+                try:
+                    logger.info(f"Disabling ETL task: {pipepline_id}")
+                    disable_task(pipepline_id)
+                except Exception as e:
+                    logger.error(f"Failed to disable task: {pipepline_id}. Error: {e}")
```

Comment:
```suggestion
                    logger.warning(f"Failed to disable task: {pipepline_id}. Error: {e}")
```
Since we've decided to suppress the error and not raise something to the user. Also since the pipline is disabled here, why don't we return early if we chose to suppress this?

Reply:
Addressed.